### PR TITLE
Fix/quest sdk no boundary

### PR DESCRIPTION
**Pull request by andybak**

### CODE REVIEW

File: Assets/Oculus/OculusProjectConfig.asset [hunk 1]
```diff
@@ -12,33 +12,34 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 05d394ae2a81edd4cbc3c51917e766e3, type: 3}
   m_Name: OculusProjectConfig
   m_EditorClassIdentifier: 
-  targetDeviceTypes: 01000000020000000300000004000000
```
Comment (mikeage) [inline Assets/Oculus/OculusProjectConfig.asset:15]: Did we remove a device here?

File: Assets/Oculus/OculusProjectConfig.asset [hunk 2]
```diff
@@ -12,33 +12,34 @@ MonoBehaviour:
...
+  targetDeviceTypes: 010000000200000003000000
   allowOptional3DofHeadTracking: 0
   handTrackingSupport: 0
   handTrackingFrequency: 0
   handTrackingVersion: 0
-  multimodalHandsControllersSupport: 0
-  anchorSupport: 1
```
Comment (mikeage) [inline Assets/Oculus/OculusProjectConfig.asset:21]: disabled?

**Review by mikeage — COMMENTED** [review]
The manifest parts look good. I know we shouldn't look too carefully at Unity files, but normally it's just additions -- here there were some things that seem to have been removed or disabled. Deliberate?

Also, I'd like to test an upload for Quest on this branch -- any objections if I push a commit, later to be reverted, to try this?

**Comment by andybak** [comment]
All the changes in the Unity files were by Unity - I didn't change anything manually except updating the package version

**Comment by andybak** [comment]
I'm going to try reverting the changes to those files locally to see if the new sdk automatically makes the same changes again.

**Comment by mikeage** [comment]
ok. Also note that the Mac failure was a real failure, but perhaps we just need to delete the package cache

File: Assets/Oculus/OculusProjectConfig.asset (hunk 2 above)
Comment (andybak) [reply Assets/Oculus/OculusProjectConfig.asset:21]: This seems better

**Comment by mikeage** [comment]
The Mac failure is real, but I think we should probably just be removing the package on Mac

**Review by mikeage — APPROVED** [review]

**Pull request merged**